bundles/pubsub/pubsub_admin_tcp/README.md

```diff
@@ -0,0 +1,112 @@
+---
+title: PSA TCP 
```

Comment:
This readme contains more info than only the PSA TCP. Can this be promoted to higher level (and maybe add the zmq.static.bind.url, etc config options.